test/tests/options.js

```diff
@@ -149,9 +149,11 @@ describe('webpack-serve Options', () => {
   t('should accept an open:Boolean option', (done) => {
     const config = load('./fixtures/basic/webpack.config.js');
     config.serve.open = true;
+    clip.writeSync('foo');
 
     serve({ config }).then(({ close }) => {
       hook = (...args) => {
+        assert.equal(clip.readSync(), 'foo');
```

Comment:
looks good 👍 

can you add a comment above this line to note that it tests that the clipboard option is disabled when open is used?